app.rb

```diff
@@ -69,8 +73,91 @@ def list_rentals(person_id = nil)
     end
   end
 
+  def load_data_from_json
+    load_books_from_json
+    load_people_from_json
+    load_rentals_from_json
+  end
```

Comment:
👏 Good job on setting up data loading in the app! 👏 However, there are a couple of areas that need attention:

- [ ] **Error Loading Rentals on Startup:** Currently, there seems to be an error when loading rentals on startup. The error message indicates an issue with converting nil to an integer at index 0. Please investigate the saved data and fix this error to ensure smooth data loading.

https://github.com/MostafaShahat2021/OOP-school-library/assets/55840999/3e588d6a-b1e8-4be7-8b10-d41163e10696


- [ ] **Consistent Person IDs for Rentals:** It appears that each time the app loads, the ID of the person is changing. This could impact the reference for rental persons. It's crucial to maintain a consistent person ID for reference, especially in the context of rentals. Verify and address any issues related to person ID changes after loading, ensuring the rental person references work seamlessly.

![image](https://github.com/MostafaShahat2021/OOP-school-library/assets/55840999/7779a7e9-b10a-4236-8cf1-3e468b84c5b4)


By resolving these issues, you'll enhance the stability and functionality of your app. Keep up the great work! 🚀📚

Reply:
Hey @shahadat3669 Thanks a lot for your suggestions, I'll fix that 👍